### ISSUE TITLE
Add a "trying out kiss" page with kiss-live and chroot info

### ISSUE THREAD

**Issue by acheam0** [kiss-community/wiki#2]

**Comment by sdsddsd1** [comment]
In what should this section differ to the install guide?
One could take the guide minus the stuff which is needed to boot on baremetal.

**Comment by acheam0** [comment]
Yeah, the chroot section could pretty much just be that. Download the
rootfs, unpack it, and kiss-chroot into it.

Kiss-live could also be mentioned, but because its not officially a KISS
thing, i'm not sure if that would be suitable for the wiki.

**Comment by dilyn-corner** [comment]
If @eudaldgr would like to write an article for `kiss-live` it would be a welcome edition! Otherwise, I could write something up. It would take me a while; I still have to reconnect with them on `KISS-kde` work there.

As for chrooting, I don't think it warrants more than the scant references it already has. There isn't a whole lot to try anyways

**Comment by eudaldgr** [comment]
What kind of article would you want? About to use `kiss-live` images or to use `kiss-live` script?

**Comment by acheam0** [comment]
I was thinking "using kiss live"
